modular_skyrat/modules/engineering_override/code/engineering_override.dm

```diff
@@ -57,19 +56,20 @@
 	var/engineering_override = FALSE
 
 /// Check for the three states of open access. Emergency, Unrestricted, and Engineering Override
-/obj/machinery/door/allowed(mob/Mob)
+/obj/machinery/door/allowed(mob/interacting_human)
 	if(emergency)
 		return TRUE
 
-	if(unrestricted_side(Mob))
+	if(unrestricted_side(interacting_human))
 		return TRUE
 
 	if(engineering_override)
-		var/mob/living/carbon/human/user = Mob
+		var/mob/living/carbon/human/user = interacting_human
```

Comment:
Swap `user` and `interacting_human` around, you've got it backwards ;)

Reply:
> plus, to make it even easier, /obj/machinery/door is already hooked into security alert level changes for some fuckin' reason you can just override `/obj/machinery/door/proc/check_security_level(datum/source, new_level)` on airlock and do all your stuff in there

Done.